app/styles/GobalStyles.ts

```diff
@@ -16,6 +16,10 @@ const GlobalStyle = createGlobalStyle`
   .formFieldset {
     border: none;
   }
+  // Respect line breaks in strings
+  label {
+    white-space: pre-line;
```

Comment:
@BryanGK check this one out, a super helpful one liner that respects line breaks in strings. I was confused how we were going to add the multi paragraph title in the benefits page text area then I remembered this one.

https://www.figma.com/file/nhOquOVmWD1P3BxEY0KIh5/CCBC-Intake?node-id=1059%3A7026

Reply:
oh nice find... wasn't sure how to do that one!